.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Description
+
+
+## Related issues
+
+* none
+
+## Related code and/or documentation pull requests
+
+* none
+
+## Documentation
+
+- [ ] User facing documentation not needed
+- [ ] Release notes entry not needed
+
+### Release notes section
+
+- [ ] General
+- [ ] Security
+- [ ] Connector(s)
+- [ ] SPI
+
+Specify which connectors are affected:
+
+
+### Release notes text
+
+<!-- provide a text suggestion here, and update in collaboration with maintainers -->
```

Comment:
This will basically NOT render in the display but help when editing.. we can do that for others as well.